hw/ip/i2c/dv/i2c_sim_cfg.hjson

```diff
@@ -120,6 +120,11 @@
       name: i2c_host_stress_all
       uvm_test_seq: i2c_host_stress_all_vseq
     }
+    {
+      name: i2c_target_smoke
+      uvm_test_seq: i2c_target_smoke_vseq
+      run_opts: ["+i2c_agent_mode=Host", "+test_timeout_ns=100_000"]
```

Comment:
The operating mode of the agent is more dynamic in nature - I will suggest not externally supplying it as a plusarg - instead, have the sequence set it.